job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext/archiveGatling.groovy

```diff
@@ -0,0 +1,13 @@
+job('example-1') {
+    publishers {
+        archiveGatling()
+    }
+}
+
+job('example-2') {
+    publishers {
+        archiveGatling() {
+            enabled(false)
+        }
+    }
+}
```

Comment:
codenarc fails likely here!
it's worth if you run `./gradlew clean check`
You can also speedup your test if you want to run only individual tests for instance:
`./gradlew -Dtest.single=PublisherContextSpec :job-dsl-core:check`